projects/media-viewer/src/lib/key-strokes/defult-key-bindings.json

```diff
@@ -0,0 +1,3 @@
+[
+  { "code": "ArrowDown", "ctrlKey": false, "altKey": true, "actionId": "NEXT_PAGE"}
```

Comment:
do we need the shift key here as well?

Reply:
It well might be the case. It's just an example.